base/applications/cmdutils/more/lang/bg-BG.rc

```diff
@@ -17,15 +17,11 @@ Options:\n\
   Command                    A command, whose output shall be displayed.\n\n\
 If extended features are enabled, the following commands are available\n\
 at ""-- Continue --"" prompt:\n\
-   P n      Display n lines.\n\
-   S n      Skip n lines.\n\
-   F        Display next file.\n\
-   Q        Quit.\n\
-   =        Show line number.\n\
-   ?        Show help line.\n\
-   <Space>  Display next page.\n\
-   <Enter>  Display next line.\n\
-   <Esc>    Same as Q.\n"
+   P n      Display n lines.        <Space>  Display next page.\n\
+   S n      Skip n lines.           <Enter>  Display next line.\n\
+   F        Display next file.      <Esc>    Same as Q.\n\
+   Q        Quit.                   =        Show line number.\n\
+   ?        Show help line.\n"
```

Comment:
Previous formatting might have been better, because it's not in all languages that these descriptions remain as short as that, and now for these languages one would need to change that formatting.

Reply:
Reverted, too.